app/views/dashboards/_actions.html.erb

```diff
@@ -14,7 +14,7 @@
         href="<%= user_series_path(current_user.username) %>"
         <%= params[:action] == "series" && (params[:which] == "organization" || params[:which].blank?) ? ' aria-current="page"'.html_safe : "" %>>
         <%= t("views.dashboard.actions.series") %>
-        <span class="crayons-indicator"><%= @user.collections.count %></span>
+        <span class="crayons-indicator"><%= @user.collections.joins(:articles).distinct.order(created_at: :desc) %></span>
```

Comment:
This is more `ActiveRecord` than I want to see in a view.  A named scope would be preferable to me. We also don't need the `order`, it should be replaced by `count`, this is what the current code renders:

![Screen Shot 2022-01-17 at 10 32 16](https://user-images.githubusercontent.com/47985/149703957-f9648366-aa78-4a85-a771-2a4190ceb2ba.png)
u

Reply:
thanks, @citizen428 for your response. I don't know ruby so I picked this snippet from the collections controller. I'll make these changes.

Reply:
This won't work either. You added a method that never gets called, so `@series_count` is `nil` which becomes an empty string in the rendered view:

![Screen Shot 2022-01-17 at 14 31 18](https://user-images.githubusercontent.com/47985/149726375-93412f26-18ed-4507-8bec-75230d1cbc31.png)

You can look at the `show` method in `DashboardsController` and potentially add the count there. 

But we probably want to introduce an `ActiveRecord` scope in the `Collection` model, so we don't have to duplicate that snippet between the two different controllers.

Since you mentioned that you don't know Ruby I pushed the change to this branch now.

Reply:
thanks @citizen428